code/modules/clothing/under/_under.dm

```diff
@@ -98,9 +97,9 @@
 		if(!alt_covers_chest)
 			body_parts_covered |= CHEST
 
-	if(mutantrace_variation && ishuman(user))
+	if(((supports_variations & DIGITIGRADE_VARIATION_NO_NEW_ICON) ||(supports_variations & DIGITIGRADE_VARIATION)) && ishuman(user))
```

Comment:
```suggestion
	if((supports_variations & (DIGITIGRADE_VARIATION|DIGITIGRADE_VARIATION_NO_NEW_ICON)) && ishuman(user))
```
At least i'm pretty sure this works.

Reply:
![image](https://user-images.githubusercontent.com/75460809/159213649-edd7f8ff-3a47-43b2-ba71-0f07a9cb34fe.png)
Apparently it does!